src/kubectlUtils.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from "vscode";

```

Comment:
I'm trying to use the `Host` interface in new modules so that tests can inject mocks and verify that e.g. errors are displayed when expected.